python/examples/atomistic/running-ase-md.py

```diff
@@ -0,0 +1,269 @@
+ax.set_xlabel("$\\AA$")
+ax.set_ylabel("$\\AA$")
+plt.show()
+
+# %%
+# And now let's make a super cell with this
+
+atoms = ase.build.make_supercell(primitive, 3 * np.eye(3))
+
+ax = ase.visualize.plot.plot_atoms(atoms, radii=0.5)
+ax.set_xlabel("$\\AA$")
+ax.set_ylabel("$\\AA$")
+
+plt.show()
+
+# %%
+#
+# We'll also need to initialize the velocities of the atoms to match our simulation
+# temperature.
+
+# The atoms start with zero velocities, hence zero kinetic energy and zero temperature
+print("before:")
+print("kinetic energy =", atoms.get_kinetic_energy(), "eV")
+print("temperature =", atoms.get_temperature(), "K")
+
+ase.md.velocitydistribution.MaxwellBoltzmannDistribution(atoms, temperature_K=300)
+print("\nafter:")
+print("kinetic energy =", atoms.get_kinetic_energy(), "eV")
+print("temperature =", atoms.get_temperature(), "K")
+
+# %%
+#
+# Running the simulation
+# ----------------------
+#
+# The final step to run the simulation is to register our model as the energy calculator
+# for these ``atoms``. This is the job of
+# :py:class:`ase_calculator.MetatensorCalculator`, which takes either an instance of
+# :py:class:`MetatensorAtomisticModel` or the path to a pre-exported model, and allow to
+# use it to compute the energy, forces and stress acting on a system.
+
+# define & wrap the model, using the initial positions as equilibrium positions
+model = HarmonicModel(k=3.14, equilibrium_positions=torch.tensor(atoms.positions))
+capabilities = ModelCapabilities(
+    length_unit="Angstrom",
+    species=[6],
+    outputs={
+        "energy": ModelOutput(quantity="energy", unit="eV", per_atom=False),
+    },
+)
+wrapper = MetatensorAtomisticModel(model.eval(), capabilities)
+
+# Use the wrapped model as the calculator for these atoms
+atoms.calc = MetatensorCalculator(wrapper)
+
+# %%
+#
+# We'll run the simulation in the constant volume/temperature thermodynamics ensemble
+# (NVT or Canonical ensemble), using a Langevin thermostat for time integration. Please
+# refer to the corresponding documentation (:py:class:`ase.md.langevin.Langevin`) for
+# more information!
+
+integrator = ase.md.Langevin(
+    atoms,
+    timestep=1.0 * ase.units.fs,
+    temperature_K=300,
+    friction=0.1 / ase.units.fs,
+)
+
+# collect some data during the simulation
+trajectory = []
+potential_energy = []
+kinetic_energy = []
+total_energy = []
+temperature = []
+
+for step in range(800):
+    # run a single simulation step
+    integrator.run(1)
+
+    # collect data about the simulation
+    potential_energy.append(atoms.get_potential_energy())
+    kinetic_energy.append(atoms.get_kinetic_energy())
+    total_energy.append(atoms.get_total_energy())
+    temperature.append(atoms.get_temperature())
+
+    if step % 10 == 0:
+        trajectory.append(atoms.copy())
+
+# %%
+#
+# We can use ASE and matplotlib to visualize the trajectory
+
+animation = ase.visualize.plot.animate(
+    trajectory, radii=0.5, interval=100, save_count=None
```

Comment:
There seems also there are some problems with the tight layout. Most likely a matplotlib or ase thing that we can't fix.

![Screenshot 2023-12-19 at 15 02 55](https://github.com/lab-cosmo/metatensor/assets/15192133/03ae9ece-2615-4e33-9c73-ecf01cd862b3)

Reply:
Okay seems like really a browser/safari issue. Works for me as well in firefox and chrome.